tests/unit/plugins/janitor/test_gpubsub_publisher.py

```diff
@@ -88,11 +88,17 @@ def gpubsub_publisher_inst(monkeypatch, kwargs):
     assert 0 == gpubsub_publisher_inst.changes_channel.qsize()
 
 
+@pytest.mark.parametrize('publish_completed', (True, False))
 @pytest.mark.asyncio
-async def test_publish(gpubsub_publisher_inst):
+async def test_publish(gpubsub_publisher_inst, publish_completed, mocker):
     """Publish received messages."""
     datetime.datetime = conftest.MockDatetime
-    msg1 = {'message': 'one'}
+    msg1 = {'action': 'additions', 'resourceRecords': {'name': 'a.b.com.'}}
```

Comment:
I know this isn't strictly part of the PR, but I'd like to see the timestamp and bytes_msg1 lines replaced with hardcoded values - otherwise we're repeating a chunk of the function.